verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -179,6 +190,11 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
             symptoms: {
                 provided: rowData.symptoms?.split(', '),
             },
+            transmission: {
+                route: rowData.transmissionRoute,
+                place: rowData.transmissionPlace,
+                linkedCaseIds: rowData.transmissionLinkedCaseIds?.split(', '),
```

Comment:
if you allow editing of these here then it is very brittle, we should split on , and then trim the remaining of the ID string otherwise someone can just remove a space and it won't be split.

Reply:
Done.